src/main/java/com/gluonhq/strange/local/SimpleQuantumExecutionEnvironment.java

```diff
@@ -75,11 +74,7 @@ public Result runProgram(Program p) {
                 int pt = 1 << pw;
                 int div = i/pt;
                 int md = div % 2;
-                if (md == 0) {
-                    probs[i] = probs[i].mul(initalpha[j]);
-                } else {
-                    probs[i] = probs[i].mul(Math.sqrt(1-initalpha[j]*initalpha[j]));
-                }
+                probs[i] = probs[i].mul( md == 0? initalpha[j]: Math.sqrt(1-initalpha[j]*initalpha[j]) );
```

Comment:
In this particular case, I prefer the clear blocked IF THEN ELSE ENDIF

Reply:
The reason I did that is 
- it is an assignment to the same variable 
- same  multiplication operation, where only second argument changes

Reply:
The bytecode will be exactly the same, so that should not be an issue.
From a mathematical point, the if {} else {} structure is just way more readable.

Reply:
Reverted